### PR TITLE
Feature/ap111 add camelize template filter

### PR DESCRIPTION
**Pull request by micahklaz**

### CODE REVIEW

File: packages/ap-frontend/docs/templatetags.rst [hunk 1]
```diff
@@ -671,3 +671,28 @@ Usage:
...
 
     {% component "@alliancesoftware/ui" "NumberInput" default_value=widget.value|none_as_nan %}{% endcomponent %}
+
+
+``camelize``
+--------------------
+
+Recursively converts dictionary keys from ``snake_case`` to ``camelCase`` for passing props to the frontend
+
+Calls :meth:`~allianceutils.util.camelize <https://github.com/AllianceSoftware/django-allianceutils?tab=readme-ov-file#camelize>`_
```
Comment (davecoates) [inline packages/ap-frontend/docs/templatetags.rst:681]: this is wrong format (see [output here](https://alliance-platform--52.org.readthedocs.build/projects/frontend/52/templatetags.html#camelize))

Should just be 

```
Calls `camelize <https://github.com/AllianceSoftware/django-allianceutils?tab=readme-ov-file#camelize>`_
```

File: packages/ap-frontend/docs/templatetags.rst [hunk 2]
```diff
@@ -671,3 +671,28 @@ Usage:
...
+.. code-block:: html+django
+
+    {% component "MyComponent" props=widget.attrs|camelize %}{% endcomponent %}
+
+.. note::
+
+    If you need to pass keys to ``ignore`` in the function call, you should run the ``camelize`` utility function in ``get_context_data`` manually,
+    rather than using this filter.
+
+.. code-block:: python
```
Comment (davecoates) [inline packages/ap-frontend/docs/templatetags.rst:694]: indent this section so appears within the note

**Comment by davecoates** [comment]
Also `yarn changeset`. can you squash commits so it's a single commit including the changeset?

File: packages/ap-frontend/docs/templatetags.rst (hunk 2 above)
Comment (micahklaz) [reply packages/ap-frontend/docs/templatetags.rst]: Fixed, thanks.

File: packages/ap-frontend/docs/templatetags.rst (hunk 1 above)
Comment (micahklaz) [reply packages/ap-frontend/docs/templatetags.rst]: Fixed, thanks.

**Pull request merged**